glean-core/src/common_metric_data.rs

```diff
@@ -45,17 +49,25 @@ impl TryFrom<i32> for Lifetime {
     }
 }
 
+/// The common set of data shared across all different metric types.
 #[derive(Default, Debug, Clone)]
 pub struct CommonMetricData {
+    /// The metric's name.
     pub name: String,
+    /// The metric's category.
```

Comment:
Is it worth adding `may contain dots to specify subcategories`?

Reply:
Hm, to me that implies that we somehow handle "sub"categories, which we don't. Even in the YAML this it not another nested object, it's just a flat name with dots.

Reply:
Yeah -- that's fine.  I don't feel too strongly about this -- particularly at this point (it's more important for folks writing metrics.yaml files).  Let's leave it as-is.